src/app/landing-page/schedule/schedule.component.ts

```diff
@@ -85,11 +85,23 @@ export class ScheduleComponent implements OnInit, OnChanges {
     })
 
     tempSchedule.forEach(day => {
-      day.sessions.Workshop.all.sort((a, b) => {
-        a.date < b.date
+      day.sessions.Keynote.sort(function(a, b) {
+        if (a.date <= b.date) return -1;
+        else return 1;
       })
-      day.sessions.Presentation.all.sort((a, b) => {
-        a.date < b.date
+
+      day.sessions.Workshop.all.sort(function(a, b) {
+        if (a.date < b.date) return -1;
+        if (a.date > b.date) return 1;
+        // Dates are equal, sort by place
+        return a.place === 'Room 1' ? -1 : 1;
```

Comment:
Não seria melhor fazer um compare entre `a.place` e `b.place`, pois caso para o ano seja `Place 1` e `Place 2` continuava a funcionar.

Algo assim:
```suggestion
        return a.place.localeCompare(b.place)
```